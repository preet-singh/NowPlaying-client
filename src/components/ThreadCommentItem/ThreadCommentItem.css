.thread-comment-item {
  display: flex;
  flex-direction: column-reverse;
  text-align: center;
  width: 100%;
  padding: 5px 20px;
  margin: auto;
  margin-bottom: 10px;

}

.thread-comment-item {
  /* border: 1px solid black; */
  padding: 0.3em;
}

.comment-time-stamp {
  font-weight: bold;
}