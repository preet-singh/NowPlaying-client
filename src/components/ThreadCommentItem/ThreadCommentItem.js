import React from 'react';
import './ThreadCommentItem.css';

function ThreadCommentItem(props) {
  return (
    <div className='comment-item'>
      <p>Comment text here</p>
      <span className='comment-time-stamp'>0:00</span>
    </div>
  )
}

export default ThreadCommentItem;