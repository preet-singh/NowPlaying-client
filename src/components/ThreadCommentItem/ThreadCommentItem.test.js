import React from 'react';
import ThreadCommentItem from './ThreadCommentItem.js';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`ThreadCommentItem component`, () => {

  it('renders the ThreadCommentItem', () => {
    const wrapper = shallow(<ThreadCommentItem />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})