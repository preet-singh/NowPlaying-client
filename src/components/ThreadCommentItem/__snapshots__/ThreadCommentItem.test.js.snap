// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThreadCommentItem component renders the ThreadCommentItem 1`] = `
<li
  className="thread-comment-item"
>
  <p>
    : 
  </p>
  <p
    className="comment-time-stamp"
  />
</li>
`;
